DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Administrator/_SearchableAdminCard.cshtml

```diff
@@ -4,12 +4,64 @@
 <div class="searchable-element nhsuk-panel" id="@Model.Id-card">
   <details class="nhsuk-details nhsuk-expander nhsuk-u-margin-bottom-0">
     <summary class="nhsuk-details__summary nhsuk-u-padding-top-0 nhsuk-u-padding-left-0">
-      <span class="nhsuk-details__summary-text searchable-element-title" id="@Model.Id-name" name="name" role="heading" aria-level="2">
+      <span class="nhsuk-details__summary-text searchable-element-title" id="@Model.Id-name" name="name">
         @Model.Name
       </span>
     </summary>
     <div class="nhsuk-details__text">
-
+      <div class="nhsuk-grid-row">
+        <div class="tags">
+          @if (!Model.IsLocked) {
+            <div class="tag nhsuk-u-font-size-14" value="@("IsLocked|false")" hidden>
+              Not locked
+            </div>
+          }
+          @foreach (var (displayText, filter) in Model.Tags) {
+            <div class="tag nhsuk-u-font-size-14" value="@filter">

```

Comment:
I wonder if we should leave adding these values until we determine that they're needed for JS to keep the MRs more concise and relevant to the ticket and to avoid forgetting to remove the code if the JS version approach ends up being different.. 🤔

Reply:
(unless you think it will be more work than worth it as I don't feel particularly strongly)

Reply:
I don't see any point removing them now I've already added them until I find they aren't needed. I think they'll be used given the approximate plan I have for the JS filtering